db/db_impl/db_impl.cc

```diff
@@ -4727,8 +4728,29 @@ Status DBImpl::CreateColumnFamilyWithImport(
   return status;
 }
 
+Status DBImpl::VerifyFileChecksums(const ReadOptions& read_options) {
+  return VerifyChecksum(read_options, /*use_file_checksum=*/true);
+}
+
 Status DBImpl::VerifyChecksum(const ReadOptions& read_options) {
+  return VerifyChecksum(read_options, /*use_file_checksum=*/false);
+}
+
+Status DBImpl::VerifyChecksum(const ReadOptions& read_options,
```

Comment:
Not a fan of overloading in derived class a function that is in base class, but not that overload.

Reply:
Sure, will rename it to `VerifyChecksumInternal()`.